### ISSUE TITLE
Add config for blocking copies of levels

### ISSUE THREAD

**Issue by Arcadius2006** [LBPUnion/ProjectLighthouse#199]
There should be an option to block publishing a level if another level that's exactly the same is published on the server

**Comment by Leafar70** [comment]
I completely agree

**Comment by turecross321** [comment]
idk man this would pretty easily be bypassed even if it was added

**Comment by TorutheRedFox** [comment]
yeah, just change one tiny little thing and the hash will be different

**Comment by Arcadius2006** [comment]
still a good thing to have regardless

**Comment by Oo1Toyo1oO** [comment]
This is actually an amazing idea.
